NGitLab.Mock/Clients/GitLabClient.cs

```diff
@@ -41,8 +41,8 @@ public GitLabClient(ClientContext context)
 
         public IGraphQLClient GraphQL
         {
-            get => _overrideGraphQLClient ?? new GraphQLClient(Context);
-            internal set => _overrideGraphQLClient = value;
+            get => _graphQLClient ??= new GraphQLClient(Context);
```

Comment:
I think we should do it for all properties or not do it. I don't think there is a reason to special-case this client.